src/components/TopBar.js

```diff
@@ -324,6 +326,21 @@ const TopBar = Utils.connectStore(authStore, 'authState')(class TopBar extends C
             style: { flex: 'none', padding: 28, marginTop: 'auto' }
           }, [
             h(CromwellVersionLink, { variant: 'light', style: { textDecoration: 'underline', color: colors.accent(0.2) } }),
+            isBioDataCatalyst() && h(Fragment, [
+              div([h(Link,
+                {
+                  style: { textDecoration: 'underline', color: colors.accent(0.2) },
+                  href: Nav.getLink('privacy'),
+                  onClick: () => this.hideNav(),
+                  ...Utils.newTabLinkProps
+                }, ['Terra Privacy Policy'])]),
+              div([h(Link, {
+                href: Nav.getLink('terms-of-service'),
+                style: { textDecoration: 'underline', color: colors.accent(0.2) },
+                onClick: () => this.hideNav(),
+                ...Utils.newTabLinkProps
```

Comment:
If we want to keep the behavior from the original footer, these should not open in a new tab since they are local links.